### PR TITLE
Added signIn and signOut functionality in UserController

### PR DESCRIPTION
**Pull request by DhanashreeChandgude**
Hi @Akkshay3,
I have added changes for sign in and sign out functionality.
Please review and merge.

### CODE REVIEW

File: quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java [hunk 1]
```diff
@@ -20,24 +28,66 @@
...
+
+        String uuid = userAuthService.signup(userEntity);
         SignupUserResponse userResponse =
                 new SignupUserResponse();
         userResponse.setId(uuid);
         userResponse.setStatus("USER SUCCESSFULLY REGISTERED");
         return new ResponseEntity<SignupUserResponse>(userResponse, HttpStatus.CREATED);
 
     }
+
```
Comment (Akkshay3) [inline quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java:58]: Add Java Doc

File: quora-service/src/main/java/com/upgrad/quora/service/business/UserAuthService.java [hunk 2]
```diff
@@ -49,5 +59,55 @@ private boolean isEmailInUse(final String email) {
...
+                jwtTokenProvider.generateToken(userEntity.getUuid(), now, expiresAt));
+        userAuthEntity.setLoginAt(now);
+        userAuthEntity.setExpiresAt(expiresAt);
+
+        userAuthRepository.save(userAuthEntity);
+        userRepository.save(userEntity);
+
+        return userAuthEntity;
+    }
+
```
Comment (Akkshay3) [inline quora-service/src/main/java/com/upgrad/quora/service/business/UserAuthService.java:112]: Add java doc for signOut

**Review by Akkshay3 — COMMENTED** [review]
Changes looks good to me, but java doc is missing . please add them

File: quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java (hunk 1 above)
Comment (DhanashreeChandgude) [reply quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java:58]: Added

File: quora-service/src/main/java/com/upgrad/quora/service/business/UserAuthService.java (hunk 2 above)
Comment (DhanashreeChandgude) [reply quora-service/src/main/java/com/upgrad/quora/service/business/UserAuthService.java:112]: Added

**Pull request merged**